### ISSUE TITLE
In Python 3.11, error occured

### ISSUE THREAD

**Issue by Remember2015** [KupynOrest/head_detector#12]
Error Messages:
```bash
File "/root/test/libs/vgg/head_detector/flame.py", line 37, in get_flame_model
  flame_model = Struct(**pickle.load(f, encoding="latin1"))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/root/test/venv/lib/python3.11/site-packages/chumpy/__init__.py", line 1, in <module>
  from .ch import *
File "/root/test/venv/lib/python3.11/site-packages/chumpy/ch.py", line 1319, in <module>
  from . import linalg
File "/root/test/venv/lib/python3.11/site-packages/chumpy/linalg.py", line 178, in <module>
  class SvdD(Ch):
File "/root/test/venv/lib/python3.11/site-packages/chumpy/linalg.py", line 181, in SvdD
  @depends_on('x')
   ^^^^^^^^^^^^^^^
File "/root/test/venv/lib/python3.11/site-packages/chumpy/ch.py", line 1203, in _depends_on
  want_out = 'out' in inspect.getargspec(func).args
                      ^^^^^^^^^^^^^^^^^^
AttributeError: module 'inspect' has no attribute 'getargspec'. Did you mean: 'getargs'?
```
I found the problem: chumpy is not compatible with Python 3.11

**Comment by KupynOrest** [comment]
We do unfortunately rely on chumpy library but as I see this issue is being fixed - https://github.com/mattloper/chumpy/pull/59

Let me know if the code works with the chumpy fix

**Comment by Remember2015** [comment]
> We do unfortunately rely on chumpy library but as I see this issue is being fixed - [mattloper/chumpy#59](https://github.com/mattloper/chumpy/pull/59)
> 
> Let me know if the code works with the chumpy fix

it works

```bash
pip install git+https://github.com/mattloper/chumpy@9b045ff5d6588a24a0bab52c83f032e2ba433e17
```

**Comment by Remember2015** [comment]
mattloper had not merged pr, but i think this issue cloud be closed
